submissions/dashakim/popup_css_html/index.html

```diff
@@ -0,0 +1,173 @@
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/account.png" class="nav__icon icon" alt="account">
+                                    <div class="popup__text text">Account</div>
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/contacts.png" class="nav__icon icon" alt="contacts">
+                                    <div class="popup__text text">Contacts</div>
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/sheets.png" class="nav__icon icon" alt="sheets">
+                                    <div class="popup__text text">Sheets</div>
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/news.png" class="nav__icon icon" alt="news">
+                                    <div class="popup__text text">News</div>
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/slides.png" class="nav__icon icon" alt="slides">
+                                    <div class="popup__text text">Slides</div>
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/earth.png" class="nav__icon icon" alt="earth">
+                                    <div class="popup__text text">Earth</div>
+                                </a>
+                            </li>
+                            <li class="popup__item hidden">
+                                <a href="#" class="nav__link">
+                                    <img src="src/keep.png" class="nav__icon icon" alt="keep">
+                                    <div class="popup__text text">Keep</div>
+                                </a>
+                            </li>
+                            <a href="#" class="additional-link hidden">Even more from Koogle</a>
+                        </ul>
+                    </div>
+                    <div class="popup__white-triangle"></div>
```

Comment:
The same story as with horizontal line: for this case better use CSS pseudo-elements `::before` and `::after`